source/DEVS-Network/DEVSServer.class.st

```diff
@@ -0,0 +1,75 @@
+Class {
+	#name : 'DEVSServer',
+	#superclass : 'DEVSAtomicComponent',
+	#instVars : [
+		'state',
+		'messages',
+		'port'
+	],
+	#category : 'DEVS-Network',
+	#package : 'DEVS-Network'
+}
+
+{ #category : 'modeling' }
+DEVSServer >> externalTransition [ 
+	| msg |
+	msg := self peekFrom: #in.
+	(msg sourceAddress = '10.0.0.1') ifTrue: [  
+	messages add: (DEVSNetworkMessage new
+		sourceAddress: msg destinationAddress;
+		destinationAddress: msg sourceAddress;
+		destinationPort: 7003;
+		payload: #bar) ].
```

Comment:
I dislike to have this hardcoded, but I guess it is just temporary. I uderstand that this is matter of different situations which you want to simulate over network. (e.g. that servers will sent messages randomly in periodic (or random time)) and you afterall want to verify that every server is in sync with other? 
But finally **I'm not sure this will work**, external transition is triggered by incoming event, but since you don't have any incoming event, it will never add a message. I guess it should be rather internal function (that would be triggered initialization of simulation?

Reply:
See - nothing is logged here. 
![obrazek](https://github.com/user-attachments/assets/2530041c-e1e9-4841-a40f-9ac1a093e4f3)

Reply:
You need to use the snippet I've sent on discord. There is nothing sent because the first message has been removed from
being hardcoded